package_manifests.json

```diff
@@ -0,0 +1,1044 @@
+[
+    {
+        "package_name": "CapitalOne DataProfiler Expectations",
+        "icon": null,
```

Comment:
@kenwade4 I'm only include this here to show you what the output of this script is (as long as Expectations are exported in the __init__.py)

Reply:
Just remember to delete this before merging

Reply:
done!